fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -1635,38 +1636,49 @@ module.exports = function(config, DB) {
                   createSession(sessionId2, session2),
                   createSession(sessionId3, session3)
                 ])
+                // Don't paralleize these, the order of them matters
+                // because they record timestamps in the db.
                 .then(function () {
-                  return P.all([
-                    insert(uid1, addr1, evB, sessionId1),
-                    insert(uid1, addr1, evA, sessionId2),
-                    insert(uid1, addr2, evA, sessionId3),
-                    insert(uid2, addr1, evA, hex32())
-                  ])
+                  return insert(uid1, addr1, evA, sessionId2).then(P.delay.bind(1))
```

Comment:
>  they're not necessary at all

Turns out they're necessary because I added tests for the ordering of `createdAt` on security events.